builder/builder.go

```diff
@@ -325,10 +328,20 @@ func (b *Builder) submitCapellaBlock(block *types.Block, blockValue *big.Int, or
 		}
 	} else {
 		go b.ds.ConsumeBuiltBlock(block, blockValue, ordersClosedAt, sealedAt, commitedBundles, allBundles, usedSbundles, &blockBidMsg)
-		err = b.relay.SubmitBlockCapella(&blockSubmitReq, vd)
-		if err != nil {
-			log.Error("could not submit capella block", "err", err, "#commitedBundles", len(commitedBundles))
-			return err
+		if isRobBlock {
+			log.Info("DEBUG: submitting rob capella block")
+			err = b.relay.SubmitRobBlockCapella(&blockSubmitReq, vd)
+			if err != nil {
```

Comment:
id pull this out of the conditional b/c it repeats in the else. just call the SubmitBlock functions in the conditional and assign to an outer err variable

Reply:
I am planning to merge this with SubmitBlock itself for the review. I initially intended to separate them for simplicity. But for review, I ll merge both the APIs. We can always make a 2nd API in the future